conf/tacc_ls5.config

```diff
@@ -0,0 +1,36 @@
+singularityDir = "$WORK/singularity_cache"
```

Comment:
I'm sure that this has come up somewhere else and been solved. @nf-core/core can anyone remember this?

Reply:
CFC e.g. uses a separate singularity cachedir for everyone at QBIC:

https://github.com/nf-core/configs/blob/master/conf/cfc.config

Not sure the `$WORK` is a good idea as users would need to have `$WORK` set accordingly before running the workflow

Reply:
I couldn't get it to pickup the `$SINGULARITY_CACHEPATH` variable, when trying to set it there. 

`$WORK` is also set for all users, they have $HOME $WORK and $SCRATCH on tacc for each user.

Reply:
I wonder if you could use a groovy function with a default to avoid the CI errors?

```groovy
def tacc_work_dir = (System.getenv("$WORK") ?: '') + '/singularity_cache'
```

Reply:
@ewels I tried this but it appears breaking up the one line broke the CI.

Reply:
I don't think that it's this code, I think it's my suggested change to move the long list into a multi-line list. The following code:

https://github.com/nf-core/configs/blob/bb7f67e2b556e1deda34b51799d654603d9c8c9f/bin/cchecker.py#L56-L58

This isn't very nice code, I'll see if I can refactor it in your PR here for you.

Reply:
Rewritten in 0eac584 to load the YAML instead of using a regex - hopefully the CI tests should now pass...